packages/account/src/Components/currency-selector/__tests__/currency-selector.spec.tsx

```diff
@@ -326,7 +340,8 @@ describe('<CurrencySelector/>', () => {
     });
 
     it('should submit the form when getCurrentStep is not passed ', async () => {
-        const new_props = { ...props };
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Instead of disabling this can we define types for **new_props**?

Reply:
Will change the implementation